vars/cloneRepo.groovy

```diff
@@ -1,16 +1,17 @@
-def call(repo, branch){
-   echo "Cloning $repo to workspace."
-   cloneDir = repo.tokenize("/").last()
-   
-   bat "mkdir $cloneDir"
-   
+def call(repo, branch, destination = ''){
+   echo "Cloning $repo"
+   cloneDir = destination + repo.tokenize("/").last()
+
+   bat "if exist \"$cloneDir\" rmdir /S /Q \"$cloneDir\""
+   bat "mkdir \"$cloneDir\""
+
    if(!branch  || branch == null){
       branch = 'master'
    }
-   
+
    dir(cloneDir){
       git url: repo, branch: branch
    }
-   
+
    return cloneDir
-}
+}
```

Comment:
Add the blank line back to the end of this file. It's part of the groovy guidelines.

Reply:
All these blank spaces are exhausting